api/k8s/v1/k8s.proto

```diff
@@ -852,6 +860,38 @@ message CreateJobResponse {
   Job job = 1;
 }
 
+message Namespace {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Namespace",
+    pattern : "{cluster}/{name}"
+  };
+
+  string cluster = 1;
+  string name = 2;
+
+  map<string, string> annotations = 3;
+  map<string, string> labels = 4;
+}
+
+message DescribeNamespaceRequest {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Namespace",
+    pattern : "{cluster}/{name}"
+  };
+
+  string clientset = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  string cluster = 2 [ (validate.rules).string = {min_bytes : 1} ];
+  string name = 3 [ (validate.rules).string = {min_bytes : 1} ];
+
+  map<string, string> labels = 4 [ (validate.rules).map.keys.string = {min_bytes : 1} ];
```

Comment:
what does this field do on the request?

Reply:
You can provide namespace specific labels to filter by just like you can for any other resource

Reply:
i figured that was the intent but when i looked it seems like the field is never read or used for that purpose in the implementation.

Reply:
Thanks for pointing out that this is a redundant field for a describe call which is trying to get a specific resource and not list based on labels. I'll clean up the other describe calls in a separate PR as all of them have the label field.